t/t1210-pull.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+#
+# Copyright (c) 2006 Yann Dirson
+#
+
+test_description='test
+
+'
+
+. ./test-lib.sh
+
+# Need a repo to clone
+test_create_repo upstream
+
+test_expect_success \
+    'Setup upstream repo, clone it, and add patches to the clone' \
+    '
+    (cd upstream && stg init) &&
+    stg clone upstream clone
+    (cd clone && git config pull.rebase false)
+    '
+
+test_expect_success \
+    'Test that pull works' \
+    '
+    (cd clone &&
+      git checkout master &&
+      stg pull
+    )
+    '
+
+test_expect_success \
+    'Test that pull without upstream setup produces friendly error' \
+    '
+    (cd clone &&
+      stg branch --create without-upstream &&
+      ( stg pull 2>&1 | ( ! grep -q "Traceback" ) ) || ! stg pull 2>&1
```

Comment:
Instead of testing for the absence of a traceback, I'd prefer we test for the presence of our expected error message.

Reply:
Agreed. Fixed